test/AutoRest.TestServer.Tests/Mgmt/TestProjects/TestProjectTests.cs

```diff
@@ -90,5 +89,104 @@ private IEnumerable<Type> FindAllContainers()
                 }
             }
         }
+
+        private IEnumerable<Type> FindAllRestOperations()
+        {
+            Type[] allTypes = Assembly.GetExecutingAssembly().GetTypes();
+
+            foreach (Type t in allTypes)
+            {
+                if (t.Name.Contains("RestOperations") && !t.Name.Contains("Tests") && t.Namespace == _projectName)
+                {
+                    yield return t;
+                }
+            }
+        }
+
+        [Test]
+        public void ValidateSubscriptionExtensions()
```

Comment:
Can we break this test into smaller chunks otherwise 1 test will fail for lots of reasons.